src/Arcus.Messaging.Tests.Integration/Fixture/ServiceBusQueueProgram.cs

```diff
@@ -0,0 +1,44 @@
+using Arcus.EventGrid.Publishing;
+using Arcus.Messaging.Tests.Workers.MessageHandlers;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Diagnostics.HealthChecks;
+using Microsoft.Extensions.Hosting;
+
+// ReSharper disable once CheckNamespace
+namespace Arcus.Messaging.Tests.Workers.ServiceBus
+{
+    public class ServiceBusQueueProgram
+    {
+        public static void Main2(string[] args)
```

Comment:
`Main2`? 🤔

Reply:
I'll rename to less strange. If I use `Main`, it's complaining about multiple entry points.

Reply:
Have to check if we can't just ignore an entry point or if the test library requires a executable.